runtime/syntax/tex.vim

```diff
@@ -745,6 +755,8 @@ if has("conceal") && &enc == 'utf-8'
     \ ['lceil'		, '⌈'],
     \ ['ldots'		, '…'],
     \ ['le'		, '≤'],
+    \ ['left|'		, '|'],
+    \ ['left\|'		, '‖'],
```

Comment:
Commenting here as well for visibility: this should be `left\\|` (double backslash), otherwise it breaks syntax highlighting of commands. Can you fix this and upstream the change?

Reply:
If there are problems with the runtime files, please contact the maintainer (dr chip) directly (see the top of the file)

Reply:
That's what I meant by "upstream".

Reply:
Hey @clason, I'm sorry that this happened – see #12504